### DISCUSSION TITLE
Reason Counter Error

### DISCUSSION THREAD

**Discussion by jongregory1** [time4tea/gopro-dashboard-overlay#106] [Q&A] [answered]
Hello,
Found this software and thought it would be cool to experiment with. Looks like I'm doing something glaringly wrong - getting a "ReasonCounter" error. Below is what my terminal puts out. What am I doing wrong?

Thanks much for help!

venv/bin/gopro-dashboard.py /Volumes/32GB/DCIM/100GOPRO/GX010003.MP4 dashboard.mp4
Starting gopro-dashboard version 0.79.0
    Timer(GPMD - Called: 1, Total: 0.01852, Avg: 0.01852, Rate: 53.99)
    Timer(extract GPS - Called: 1, Total: 0.05644, Avg: 0.05644, Rate: 17.72)
    Timer(extract ACCL - Called: 1, Total: 0.27253, Avg: 0.27253, Rate: 3.67)
    Timer(extract GRAV - Called: 1, Total: 0.15921, Avg: 0.15921, Rate: 6.28)
    Timer(extract CORI - Called: 1, Total: 0.28438, Avg: 0.28438, Rate: 3.52)
Timer(parsing - Called: 1, Total: 0.79160, Avg: 0.79160, Rate: 1.26)
Timer(loading timeseries - Called: 1, Total: 1.42810, Avg: 1.42810, Rate: 0.70)
Timer(program - Called: 1, Total: 1.42816, Avg: 1.42816, Rate: 0.70)
Traceback (most recent call last):
  File "/Users/jon/venv/bin/gopro-dashboard.py", line 177, in <module>
    if counter.total() > 0:
AttributeError: 'ReasonCounter' object has no attribute 'total'

**Comment by time4tea** [comment, marked as answer]
Hi - you are probably running with python lower than 3.10 - the software needs 3.10.

It doesn't actually check the python version, so it's not a very good error message. I'll  improve this.

Try with python 3.10 or higher.

Thanks for your interest!

**Reply by time4tea** [reply]
I also noticed you are using an older version of the software. Please do update. You can do this by simply removing the `venv` directory and then following the installation instructions again.
The software gets updated often!

**Reply by jongregory1** [reply]
Thank you! That resolved the issue!
